nemo/collections/common/tokenizers/tokenizer_utils.py

```diff
@@ -60,38 +61,56 @@ def get_bert_special_tokens(bert_derivative):
 
 
 def get_tokenizer(
-    tokenizer_name,
-    pretrained_model_name,
-    tokenizer_model=None,
-    special_tokens=None,
-    vocab_file=None,
-    do_lower_case=False,
+    tokenizer_name: str,
+    data_file: Optional[str] = None,
+    tokenizer_file: Optional[str] = None,
+    sample_size: Optional[int] = None,
+    pretrained_model_name: Optional[str] = None,
+    special_tokens: Optional[List[str]] = None,
+    vocab_file: Optional[str] = None,
+    vocab_size: Optional[int] = None,
+    do_lower_case: Optional[bool] = False,
 ):
     '''
     Args:
     tokenizer_name: sentencepiece or nemobert
-    pretrained_mode_name ('str'): name of the pretrained model, for example: bert-base-cased
-    tokenizer_model (path): only used for sentencepiece tokenizer
-    special_tokens (dict): dict of special tokens (Optional)
-    vocab_file (str): path to vocab file
-    do_lower_case (bool): (whether to apply lower cased) - only applicable when tokenizer is build with vocab file
+    data_file: data file used to build sentencepiece
+    tokenizer_file: tokenizer model file of sentencepiece
+    sample_size: sample size for building sentencepiece
+    pretrained_model_name: name of the pretrained model from the hugging face list or 'megatron',
+        for example: bert-base-cased
+        To see the list of pretrained models, use: nemo_nlp.nm.trainables.get_bert_models_list()
+    tokenizer_model: only used for sentencepiece tokenizer
```

Comment:
oh, you have tokenizer_model as well, is it left from the old implementation?

Reply:
that was mistake deleted this line